src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -508,14 +510,15 @@ export class RelayFeeCalculator {
       gasUnits,
       tokenGasCost
     );
-    const gasFeeTotal = gasFeePercent.mul(amountToRelay).div(fixedPointAdjustment);
+    const outToInDecimals = ConvertDecimals(outputTokenInfo.decimals, inputTokenInfo.decimals);
+    const gasFeeTotal = gasFeePercent.mul(outToInDecimals(amountToRelay.toString())).div(fixedPointAdjustment);
     const capitalFeePercent = this.capitalFeePercent(
       amountToRelay,
-      token.symbol,
+      inputTokenInfo.symbol,
       deposit.originChainId.toString(),
       deposit.destinationChainId.toString()
     );
-    const capitalFeeTotal = capitalFeePercent.mul(amountToRelay).div(fixedPointAdjustment);
+    const capitalFeeTotal = capitalFeePercent.mul(outToInDecimals(amountToRelay.toString())).div(fixedPointAdjustment);
```

Comment:
wdyt about renaming `amountToRelay` to `outputAmount` ? That makes it absolutely clear that it's set in `outputTokenInfo.decimals`.

Reply:
Good call. That might be clearer.